### PR TITLE
target w/index

### PR DESCRIPTION
**Pull request by t-romani**

### CODE REVIEW

File: app/controllers/api/v1/targets_controller.rb [hunk 1]
```diff
@@ -9,6 +9,10 @@ def create
         render :show
       end
 
+      def index
+        @targets = user.targets.index
```
Comment (gonzaM) [inline app/controllers/api/v1/targets_controller.rb]: What is `.index` for?

**Review by gonzaM — COMMENTED** [review]
Good job!

File: app/controllers/api/v1/targets_controller.rb (hunk 1 above)
Comment (t-romani) [reply app/controllers/api/v1/targets_controller.rb]: I guess it is not needed then 😆

File: spec/request/target/index_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'GET #index target', type: :request do  
+  let!(:user)     { create(:user) }
+  let!(:targets)  { create_list(:target, 2, user: user) }
+  let(:data)      { parsed_data }
```
Comment (gonzaM) [inline spec/request/target/index_spec.rb]: where are you using this? 🤔

**Review by gonzaM — APPROVED** [review]

File: spec/request/target/index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  context 'when valid' do
+    let!(:auth_headers) { auth_user_headers }
+
+    it 'gets a successfull answer' do
+      expect(response).to be_successful
+    end
+
+    it 'returns list of targets' do
+      expect(parsed_data['targets'].count).to eq(targets.count)
+    end
```
Comment (JuanfraM) [inline spec/request/target/index_spec.rb:23]: The title of the spec is incorrect here, you are testing the correct amount of target.
I'd also test that the `ids` are the same as expected using `match_array `

**Review by JuanfraM — APPROVED** [review]

**Pull request merged**